### PR TITLE
Add kernel checking for RNG checkpoint

### PR DESCRIPTION
**Pull request by xiaodwan**
Virtio RNG supports from kernel-2.6.26, check the kernel version of
VM before add RNG checkpoint.

Reference:
[1] https://wiki.qemu.org/Features/VirtIORNG

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.6_5.linux.5_11.arch_x86_64.it_vddk: PASS (3416.13 s)
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_ovirt.xen.vm.hvm.linux.5_11.arch_x86_64.raw_f.NFS.rhv_upload.rhv_direct.rhv_verifypeer.nopreallocated.it_default: PASS (2963.77 s)

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_ovirt.esx.vm.6_7.linux.latest7.arch_x86_64.raw_f.NFS.rhv_upload.rhv_direct.rhv_verifypeer.nopreallocated.it_vddk: PASS (2336.60 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh Please help to review this PR, thanks.

File: provider/v2v_vmcheck_helper.py [hunk 1]
```diff
@@ -110,15 +111,19 @@ def run(self):
             logging.warn("Unspported os type: %s", self.os_type)
         return self.errors
 
-    def compare_version(self, compare_version):
+    def compare_version(self, compare_version, real_version=None, cmd=None):
         """
-        Compare virt-v2v version against given version.
+        Compare version against given version.
+
+        :param compare_version: The least version to compare
```
Comment (dzhengfy) [inline provider/v2v_vmcheck_helper.py]: Not sure what is 'least version'. Could you help refine the words?

File: provider/v2v_vmcheck_helper.py [hunk 2]
```diff
@@ -110,15 +111,19 @@ def run(self):
...
+        :param real_version: The real version to compare
+        :param cmd: the command to get the real version
         """
```
Comment (dzhengfy) [inline provider/v2v_vmcheck_helper.py]: Please add the docstring for returned value.

Comment (xiaodwan) [reply provider/v2v_vmcheck_helper.py]: I'll update them in next patch.

File: provider/v2v_vmcheck_helper.py (hunk 1 above)
Comment (xiaodwan) [reply provider/v2v_vmcheck_helper.py]: ok

**Pull request merged**